app/lib/widgets/phone_widget.dart

```diff
@@ -211,23 +211,24 @@ class PhoneAlertDialogState extends State<PhoneAlertDialog> {
 
     if (Globals().tooManySmsAttempts &&
         Globals().lockedSmsUntil > currentTime) {
+      if (!mounted) return;
       Globals().sendSmsAttempts = 0;
       showDialog(
         context: context,
-        builder: (BuildContext context) {
-          return AlertDialog(
-            title: Text(
-                'Too many attempts please wait ${((Globals().lockedSmsUntil - currentTime) / 1000).round()} seconds.'),
-            actions: <Widget>[
-              TextButton(
-                child: const Text('OK'),
-                onPressed: () {
-                  Navigator.of(context).pop();
-                },
-              ),
-            ],
-          );
-        },
+        builder: (BuildContext childContext) => CustomDialog(
+          image: Icons.info,
+          title: 'Too many attempts',
+          description:
+              'Please wait ${((Globals().lockedSmsUntil - currentTime) / 1000).round()} seconds.',
+          actions: <Widget>[
+            TextButton(
+              child: const Text('Ok'),
+              onPressed: () {
+                Navigator.pop(childContext);
+              },
+            ),
+          ],
+        ),
       );
       return;
```

Comment:
Same here. I get too many SMS attempts if I added a number without verifying it, then clicking verify button. At the same time, if I clicked back to homepage, then enterred, the sms cool down will work and a message will be sent.

Reply:
https://github.com/user-attachments/assets/d6782811-4477-4dd0-8bfa-a4c029532301